lib/src/components/NxDropdown/NxDropdown.scss

```
@@ -13,6 +13,10 @@
   display: inline-block;
   position: relative;
   width: nx-dropdown-variables.$dropdown-width;
+
+  .nx-icon:not(.nx-icon--colorful):not(.nx-dropdown-link .nx-icon):not(.nx-dropdown .disabled .nx-icon) {
```

This can be made more compact with commas instead of successive `:not`s
```suggestion
  .nx-icon:not(.nx-icon--colorful, .nx-dropdown-link .nx-icon, .nx-dropdown .disabled .nx-icon) {
```

But since we're handling disabled here, we should probably handle disabling by attribute as well
```suggestion
  .nx-icon:not(.nx-icon--colorful, .nx-dropdown-link .nx-icon, .nx-dropdown :is(.disabled, [disabled]) .nx-icon) {
```

Note that I've never used `:is` before, and am suggesting this without testing it first :slightly_smiling_face:.  Let me know if it doesn't work.

---

It worked, thanks!